source/reference/minio-mc/mc-event-remove.rst

```diff
@@ -11,17 +11,20 @@
    :depth: 2
 
 .. mc:: mc event remove
+.. mc:: mc event rm
 
 Syntax
 ------
 
 .. start-mc-event-remove-desc
 
-The :mc:`mc event remove` command removes event notification triggers on a
+The :mc:`mc event rm` command removes event notification triggers on a
 bucket.
 
 .. end-mc-event-remove-desc
 
+The :mc:`mc event remove` alias has equivalent functionality to :mc:`mc event rm`.
```

Comment:
```suggestion
The :mc:`mc event remove` command has equivalent functionality to :mc:`mc event rm`.
```

We use `alias` for other purposes, and I wouldn't confuse it by using it here in that way.